### PR TITLE
Refactoring: improve ide usage clarity

### PR DESCRIPTION
**Pull request by dougg0k**
I wanted to improve the clarity on the ide feature.

### CODE REVIEW

**Comment by dougg0k** [comment]
We could also have graphiql as the default option, without having to set anything in the ide.

File: index.d.ts [hunk 1]
```diff
@@ -74,9 +74,9 @@ declare namespace FastifyGQL {
       }
     },
     /**
-     * Serve GraphiQL on /graphiql if true or 'graphiql', or GraphQL IDE on /playground if 'playground' if routes is true
+     * Serve GraphiQL on /graphiql if 'graphiql', or GraphQL IDE on /playground if 'playground' and if routes is true
      */
-    graphiql?: boolean | string,
```
Comment (mcollina) [inline index.d.ts]: do not remove this from the types

Comment (dougg0k) [reply index.d.ts]: Leave both then?

Comment (mcollina) [reply index.d.ts]: yes, the option is still there

**Pull request merged**